Formula/r/rabbitmq.rb

```diff
@@ -58,7 +58,16 @@ def install
 
   def caveats
     <<~EOS
-      Management Plugin enabled by default at http://localhost:15672
+      Management UI is enabled by default at http://localhost:15672.
+
+      To enable all feature flags, run
+
+      rabbitmqctl enable_feature_flag all
```

Comment:
So why not just run this in a post install block?

Reply:
Because the CLI command requires the node to be running and that's not the case after installation https://github.com/Homebrew/homebrew-core/pull/158203#issuecomment-1869129510

Reply:
So far the most pragmatic option seems to be running the command in a service startup block but I have not found examples that do that and it seems to be a very hacky way to go about it.